src/webcore/executor.rs

```diff
@@ -178,6 +178,10 @@ impl EventLoopInner {
     // Reclaim space from the queue if it's going to waste
     fn shrink_if_necessary(&self) {
         let mut queue = self.microtask_queue.borrow_mut();
+        if queue.capacity() <= INITIAL_QUEUE_CAPACITY {
```

Comment:
This doesn't seem right. Let's say the queue has a capacity of 50 but a length of 1. This check will return false, but then later it calls `shrink_to_fit` which resizes it to a capacity of 1